app/models/configuration_script_source.rb

```diff
@@ -0,0 +1,6 @@
+class ConfigurationScriptSource < ApplicationRecord
```

Comment:
I thought we *weren't* going to rename this class and just leave it as ConfigurationScript.  Otherwise you will have to change it everywhere in the current application.